.github/workflows/ci.yaml

```diff
@@ -118,13 +106,6 @@ jobs:
         icacls C:\Windows\System32\bash.exe /grant administrators:F
         del C:\Windows\System32\bash.exe
 
-    - name: Install Rust Toolchain Components
-      uses: actions-rs/toolchain@v1
-      with:
-        components: clippy, rustfmt
-        override: true
```

Comment:
How can we communicate override to the Rust Toolchain?

Reply:
The `override` part? I'm not 100% sure it's being used, so we could just see if it works without it.

Reply:
The `ci` job is green ✅